### DISCUSSION TITLE
Extra white spaces before and after inputs in the input fields shows invalid message

### DISCUSSION THREAD

**Discussion by dgupta0** [jdwilkin4/contrast-ratio-repo#146] [General]
Hi, everyone! 

We have realised that when  colors are typed in any of the color format, which is part of the app functionality, then the app shows invalid input. For example - `pink ` instead of `pink` (notice the extra space at the end for first pink) will show invalid input message. This is happening for all the format, we have decided to work with. 

Should we fix it now or after release.

**Comment by jdwilkin4** [comment]
It seems like it should be an easy enough fix to use `trim()` for the user input to remove the extra whitespace.
If it seems like it will be more trouble than it is worth, then we can add it to the next release, but I would be surprised if this wasn't a quick fix.

**Comment by AndreyDmitriev39r** [comment]
Let's go with issue for that I guess.
